### PR TITLE
Finalizing tracker integration

### PR DESCRIPTION
**Pull request by Doralitze**
After the autotrack software has been included, some integration tasks are still missing.
 - [x] fix label import
 - [x] save tracker configuration in filter properties
 - [x] Move controller behavior from detection tab to controller
 - [x] Integrate instantiation of filters
 - [x] Implement filter update based on detection results
 - [x] Implement filter configuration widget that just displays the settings tab
 - [x] Test behavior with multiple trackers -> Not yet supported
 - [x] Create text file containing licenses of other open source dependencies that can also be displayed within the software ('About' Menu-Entry)
 - --Write manual entry in documentation repository-- Let's do this later.
 - [x] Implement storing and loading of tracker settings
  - [ ] Test storing and loading of tracker settings

### CODE REVIEW

File: src/controller/autotrack/Helpers/AutoTrackerSettings.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import logging
```
Comment (maxkaussow) [inline src/controller/autotrack/Helpers/AutoTrackerSettings.py:1]: nur "from logging import getLogger"
 importieren das macht es einfacher das nicht ausversehen doch logging.error() oder so aufgrufen wird

File: src/controller/autotrack/Helpers/AutoTrackerSettings.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import logging
+
+from controller.autotrack.Calibration.MappingCalibration import MappingCalibration
+from controller.autotrack.LightController import LightController
+from model.virtual_filters import AutoTrackerFilter
+
+logger = logging.Logger(__file__)
```
Comment (maxkaussow) [inline src/controller/autotrack/Helpers/AutoTrackerSettings.py:7]: und dann logger = getLogger(__name__)

File: src/controller/autotrack/Helpers/AutoTrackerSettings.py (hunk 1 above)
Comment (Doralitze) [reply src/controller/autotrack/Helpers/AutoTrackerSettings.py]: This is now fixed.

File: src/controller/autotrack/Helpers/AutoTrackerSettings.py (hunk 2 above)
Comment (Doralitze) [reply src/controller/autotrack/Helpers/AutoTrackerSettings.py]: This is now fixed.

**Review by maxkaussow — APPROVED** [review]
passt

**Pull request merged**